cypress/e2e/tests/pages/explorer/api/api-services.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { APIServicesPagePo } from '@/cypress/e2e/po/pages/explorer/api-services.po';
+import SortableTablePo from '@/cypress/e2e/po/components/sortable-table.po';
+
+describe('Cluster Explorer', () => {
+  beforeEach(() => {
+    cy.login();
+  });
+
+  describe('API: APIServices', () => {
+    let apiServicesPage: APIServicesPagePo;
+
+    beforeEach(() => {
+      apiServicesPage = new APIServicesPagePo('local');
+      apiServicesPage.goTo();
+    });
+
+    it('Should have a title', () => {
+      cy.get('h1').should('contain', 'APIServices');
```

Comment:
When `BaseResourceList` is wired in the `cy.get('h1')` part should be a getter in `ResourceListMastheadPo` (as it's a generic header in the resource list's masthead). It's then available for everyone else to use (i think alex is doing the same thing atm)